src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,18 @@
+.TodoInfo {
+  width: max-content;
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
```

Comment:
Consider using variables for border and background colors to avoid hardcoding. 💡